maven-plugin/src/main/java/org/openmrs/maven/plugins/BuildDistro.java

```diff
@@ -371,8 +407,7 @@ private void writeTemplatedFile(File targetDirectory, String distro, String vers
 		if (!compose.exists()) {
 			try (InputStream inputStream = composeUrl.openStream(); FileWriter composeWriter = new FileWriter(compose)) {
 				String content = IOUtils.toString(inputStream, StandardCharsets.UTF_8);
-				content = content.replaceAll("<distro>", distro);
-				content = content.replaceAll("<tag>", version);
```

Comment:
Can you throw in some light as to why you made this particular change?

Reply:
In the new docker-compose file we used version is formated like {TAG:-nightly}.